### PR TITLE
How to use rules article added

### PR DESCRIPTION
**Pull request by Astarsys**
- added the article
- the page added to pages.yml
- little fixes in rule.md for examples and tables. Also, added the field "emergency" to the rule object and its description.

### CODE REVIEW

File: docs/backend-api/how-to/how-to-create-a-rule.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+description: Rules used to set up conditions according to which the system logs the events and sends notifications to user.
+---
+
+# How to use rules
+
+Rules used to set up conditions according to which the system logs the events and sends notifications to user.
+
+When a server receives a new portion of data from the device, it checks whether the conditions set are true 
+or false for this data. If they are true, the server generates an event in history, logs it and immediately sends SMS or
+ Email, as specified.
```
Comment (fenik17) [inline docs/backend-api/how-to/how-to-create-a-rule.md:12]: ...or push message and save event in history

File: docs/backend-api/how-to/how-to-create-a-rule.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+Let's create a rule with conditions according to which the platform will generate events and schedule intervals when this
+ rule should work. The user must have access to rule update.
+
+#### parameters
+
+| name | description | type |
+| :------ | :------ | :----- |
+| name | The name of created rule. | string |
+| description | Rule's description. | string |
+| zone_ids | List of zones to bind where the rule will work. Leave it empty if rule should work everywhere. | array of int |
```
Comment (fenik17) [inline docs/backend-api/how-to/how-to-create-a-rule.md:25]: Parameter `zone_ids` is not allowed for rule `offline` and required for `route` and `inoutzone`.

File: docs/backend-api/how-to/how-to-create-a-rule.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+
+# How to use rules
+
+Rules used to set up conditions according to which the system logs the events and sends notifications to user.
+
+When a server receives a new portion of data from the device, it checks whether the conditions set are true 
+or false for this data. If they are true, the server generates an event in history, logs it and immediately sends SMS or
+ Email, as specified.
+ 
+### Create
```
Comment (fenik17) [inline docs/backend-api/how-to/how-to-create-a-rule.md:14]: Maybe not needed to copy&paste the full description of the resource calls in the how-to article? Just add links to API reference.

**Review by fenik17 — CHANGES_REQUESTED** [review]

File: docs/backend-api/how-to/how-to-create-a-rule.md (hunk 3 above)
Comment (Astarsys) [reply docs/backend-api/how-to/how-to-create-a-rule.md:14]: You're right. Thank you. I have changed the article. It seems much better.

File: docs/backend-api/how-to/how-to-create-a-rule.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+* `trackers` - An array of int. List trackers' IDs. Trackers which do not exist, owned by other user or deleted ignored without errors.
+ 
+API requests:
+ 
+=== "Bind"
+
+ ```shell
+ curl -X POST '{{ extra.api_example_url }}/tracker/rule/bind' \
+     -H 'Content-Type: application/json' \
+     -d '{"hash": "a6aa75587e5c59c32d347da438505fc3", "rule_id": "123", "trackers": [265489]}'
```
Comment (fenik17) [inline docs/backend-api/how-to/how-to-create-a-rule.md:74]: `"rule_id": 123`

File: docs/backend-api/how-to/how-to-create-a-rule.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+ ```
+
+=== "Unbind"
+
+ ```shell
+ curl -X POST '{{ extra.api_example_url }}/tracker/rule/unbind' \
+     -H 'Content-Type: application/json' \
+     -d '{"hash": "a6aa75587e5c59c32d347da438505fc3", "rule_id": "123", "trackers": [265489]}'
```
Comment (fenik17) [inline docs/backend-api/how-to/how-to-create-a-rule.md:82]: `"rule_id": 123`

**Review by fenik17 — CHANGES_REQUESTED** [review]

**Review by fenik17 — APPROVED** [review]

**Pull request merged**